.circleci/config.yml

```diff
@@ -6,4 +6,13 @@ jobs:
     working_directory: /go/src/github.com/hashicorp/envconsul
     steps:
       - checkout
-      - run: go test -v
+      - restore_cache:
+          keys:
+            - ct-modcache-v1-{{ checksum "go.mod" }}
+      - run: |
+          export GO111MODULE=on
```

Comment:
This could be pulled up into an [`environment`](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-job) block in CircleCI config to be a little cleaner.

Reply:
That is cleaner. I wasn't thinking I'd need it long due to it not being required with go1.13, so I didn't give it much thought. I'll move it.

Reply:
One note about this. You have to quote "on" in the environment setting section, otherwise circleci converts it to "true".